notebooks/niriss_ami_binary/2_niriss_ami_binary.ipynb

```diff
@@ -11,19 +11,21 @@
    "cell_type": "markdown",
```

Comment:
<p>Following the earlier comment about <code>pathlib.Path()</code>, I recommend changing to <code>currentdir = Path('.')</code>.</p><p> </p><p>As long as one of the objects involved is a <code>Path</code>, you can chain together composite path with <code>/</code>, so I recommend doing that over concatenating strings. In the rest of the cell, this would look like:</p><pre class="ql-syntax" spellcheck="false">inflightdata = currentdir / 'niriss_ami_binary2_inflight'
datafiles = list(sorted(inflightdata.glob('jw*uncal.fits')))
for i in datafiles:
&nbsp;&nbsp;&nbsp; print(i)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Done, thanks!